packages/kuma-gui/src/app/services/views/MeshServiceDetailView.vue

```diff
@@ -84,14 +84,16 @@
               #body
             >
               <template v-if="props.data.spec.ports.length">
-                <KumaPort
-                  v-for="connection in props.data.spec.ports"
-                  :key="connection.port"
-                  :port="{
-                    ...connection,
-                    targetPort: undefined,
-                  }"
-                />
+                <XTruncate>
+                  <KumaPort
+                    v-for="connection in props.data.spec.ports"
+                    :key="connection.port"
+                    :port="{
+                      ...connection,
+                      targetPort: undefined,
+                    }"
+                  />
+                </XTruncate>
```

Comment:
If you wrap this in an `<XLayout type="separated" truncate>` if will truncate and also fix the ever so tiny spacing discrepancy (same with MMZ).

I 'think' then we don't need a `XTruncate` component?